diff -uprN a/config.def.h b/config.def.h
--- a/config.def.h	2011-12-20 00:02:46.000000000 +0900
+++ b/config.def.h	2013-06-18 02:50:41.090173519 +0900
@@ -12,6 +12,7 @@ static const unsigned int borderpx  = 1;
 static const unsigned int snap      = 32;       /* snap pixel */
 static const Bool showbar           = True;     /* False means no bar */
 static const Bool topbar            = True;     /* False means bottom bar */
+static const int marginbottom       = 20;       /* Bottom margin for dzen */
 
 /* tagging */
 static const char *tags[] = { "1", "2", "3", "4", "5", "6", "7", "8", "9" };
diff -uprN a/dwm.c b/dwm.c
--- a/dwm.c	2011-12-20 00:02:46.000000000 +0900
+++ b/dwm.c	2013-06-18 02:50:41.090173519 +0900
@@ -1841,6 +1841,9 @@ updatebarpos(Monitor *m) {
 	m->wh = m->mh;
 	if(m->showbar) {
 		m->wh -= bh;
+		m->wh -= marginbottom;
+		if(!m->topbar)
+			m->wy += marginbottom;
 		m->by = m->topbar ? m->wy : m->wy + m->wh;
 		m->wy = m->topbar ? m->wy + bh : m->wy;
 	}
